front/public/swagger.json

```diff
@@ -3186,7 +3378,7 @@
           "userFavorite": {
             "type": "boolean",
             "description": "Status of the user favorite for this configuration",
-            "example": "true"
+            "example": "all_users"
```

Comment:
A boolean set to "all_users"??

Reply:
interesting indeed :-)  the value was changed a few days ago in swagger.json only, not in swagger_schemas.ts (https://github.com/dust-tt/dust/pull/8384/files#diff-414df243af60f5fcf7877d5d74f488a181d6b91d2b572e2256b0ac100564cec5 ) . Fixing